lib/testUtils/__tests__/createRuleTester.test.js

```diff
@@ -0,0 +1,45 @@
+"use strict"
+
+const createRuleTester = require("../createRuleTester")
+
+describe("createRuleTester", () => {
+  it("is possible to create a tester", () => {
```

Comment:
Sure, I can split them up. I just seem to have been the first that ever introduced a test for `createRuleTester` and the guidelines said to provide a test that exemplifies the issue ;) Happy to add another one before, they are quite verbose however as it pretty much uses the whole stylelint suite, but I agree it would be good to have them.